tpch/execute.py

```diff
@@ -92,15 +95,19 @@ def execute_query(query_id: str) -> None:
     expected = pl.read_parquet(DATA_DIR / f"result_{query_id}.parquet")
 
     for backend, (native_namespace, kwargs) in BACKEND_NAMESPACE_KWARGS_MAP.items():
-        if backend == "duckdb" and query_id in DUCKDB_SKIPS:
-            print(f"\nSkipping {query_id} for DuckDB")  # noqa: T201
+        if backend in {"duckdb", "sqlframe"} and query_id in DUCKDB_SKIPS:
+            print(f"\nSkipping {query_id} for {backend}")  # noqa: T201
```

Comment:
https://github.com/narwhals-dev/narwhals/issues/2226 would address this for sqlframe